docs/resources/partner_interconnect_attachment.md

```diff
@@ -0,0 +1,54 @@
+---
+page_title: "DigitalOcean: digitalocean_partner_interconnect_attachment"
+subcategory: "Networking"
+---
+
+# digitalocean_partner_interconnect_attachment
+
+-> Partner Interconnect Attachment is currently in alpha. If you are not a member of the alpha group for this feature, you will not be able to use it until is has been more widely released. Please follow the official [DigitalOcean changelog](https://docs.digitalocean.com/release-notes/) for updates.
```

Comment:
Going forward, we've been instructed to not use terms like "alpha." Instead we need to use the product lifecycle nomenclature. In this case, we probably want "private preview."

https://docs.digitalocean.com/platform/product-lifecycle/